include/llvm/Transforms/Tapir/TapirUtils.h

```diff
@@ -25,7 +25,6 @@
 #include "llvm/Transforms/Utils/ValueMapper.h"
 
 namespace llvm {
```

Comment:
Was there a reason these things were moved from a tapir namespace? I'm personally leaning towards keeping it like that.

Reply:
After reorganizing the CilkABI code to deal with the first two goals, I observed that Tapir/LLVM would fail to build on old versions of GCC.  The problem stems from a bug in how GCC handles namespaces.  This GCC bug was not fixed until recently, i.e., GCC version 7.  The workaround I found involves removing the Tapir namespace.  Given that a lot of people who want to build Tapir still use older versions of GCC, I believe we need to accommodate those older GCC versions.

Reply:
I'm also not clear on why the Tapir namespace was added in the first place.  It seemed to be pushed to the codebase in a commit that never received review.  It also added substantial clutter and redundancy to Tapir-related code and to parts of the codebase that interact with Tapir.  Namespaces are used elsewhere in LLVM, but pretty sparingly, from what I see, and in ways that add clarity, not clutter.  I don't feel the Tapir namespace accomplished that.